src/pages/home/showcase-products.jsx

```diff
@@ -0,0 +1,36 @@
+import { Link } from "react-router-dom";
+
+const ProductShowCaseCard = ({ showCaseProducts }) => {
+  return (
+    <>
+      {showCaseProducts.map((item) => {
+        return (
+          <Link 
+            to="/ProductsPage"
+            className="card-body-md card-horizontal-md home-product-card"
+            key={item._id}
+          >
+            <div className="card-img flex-align-centre ">
+              <img
+                src={item.imgsource}
+                alt="demoProducts"
```

Comment:
The alt tag should convey a more understandable message :)

Reply:
ya,  I will change this.

Thank for reviewing Tejas